Source/RMG/UserInterface/Widget/RomBrowserWidget.cpp

```diff
@@ -128,23 +128,26 @@ RomBrowserWidget::RomBrowserWidget(QWidget *parent) : QStackedWidget(parent)
     this->gridViewWidget->setVerticalScrollMode(QAbstractItemView::ScrollPerPixel);
     this->gridViewWidget->verticalScrollBar()->setSingleStep(15);
     this->gridViewWidget->setWordWrap(true);
-    this->gridViewWidget->setContextMenuPolicy(Qt::ContextMenuPolicy::CustomContextMenu);
     this->gridViewWidget->setFrameStyle(QFrame::NoFrame);
     int iconWidth = CoreSettingsGetIntValue(SettingsID::RomBrowser_GridViewIconWidth);
     int iconHeight = CoreSettingsGetIntValue(SettingsID::RomBrowser_GridViewIconHeight);
     this->gridViewWidget->setIconSize(QSize(iconWidth, iconHeight));
     this->addWidget(this->gridViewWidget);
     connect(this->gridViewWidget, &QListView::doubleClicked, this, &RomBrowserWidget::on_DoubleClicked);
-    connect(this->gridViewWidget, &QListView::customContextMenuRequested, this, &RomBrowserListViewWidget::customContextMenuRequested);
     connect(this->gridViewWidget, &QListView::iconSizeChanged, this, &RomBrowserWidget::on_gridViewWidget_iconSizeChanged);
     connect(this->gridViewWidget, &Widget::RomBrowserGridViewWidget::ZoomIn, this, &RomBrowserWidget::on_ZoomIn);
     connect(this->gridViewWidget, &Widget::RomBrowserGridViewWidget::ZoomOut, this, &RomBrowserWidget::on_ZoomOut);
 
     // configure context menu policy
-    this->setContextMenuPolicy(Qt::ContextMenuPolicy::CustomContextMenu);
-    connect(this, &QStackedWidget::customContextMenuRequested, this, &RomBrowserWidget::customContextMenuRequested);
+    this->setContextMenuPolicy(Qt::CustomContextMenu);
+    connect(this, &QStackedWidget::customContextMenuRequested, this, &RomBrowserWidget::mainContextMenuRequested);
 
-    // configure context menu actions
+    // define context menus
+    this->mainContextMenu = new QMenu(this);
+    this->headerContextMenu = new QMenu(this);
```

Comment:
This is unrequired

Reply:
Are you sure? Deleting the context menu definitions causes RMG to crash for me whenever the context menu is opened.

Reply:
After my overlook, this is now fine, I'll mark it as resolved!